.github/RELEASE.md

```diff
@@ -1,37 +1,12 @@
 # 🌟 Features
 
-- Official Github releases (https://github.com/IntelLabs/kafl.fuzzer/pull/31)
-
 # ✨ Improvements
 
-- turn hardcoded serial and append params into default config values (https://github.com/IntelLabs/kafl.fuzzer/pull/7)
-- Smarter CPU affinity selection (https://github.com/IntelLabs/kafl.fuzzer/pull/10)
-- config.py: do variable expansion for qemu_base and qemu_extra options (https://github.com/IntelLabs/kafl.fuzzer/pull/19)
-- qemu.py: redirect qemu outputs on --log (https://github.com/IntelLabs/kafl.fuzzer/pull/29)
-
 # 🔧 Fixes
 
-- abort if both --resume and --purge are given (https://github.com/IntelLabs/kafl.fuzzer/pull/2)
-- fix infinite loop in kafl_debug / gdb action (https://github.com/IntelLabs/kafl.fuzzer/pull/3)
-- worker.py: fix custom timeout setting in execute_naked() (https://github.com/IntelLabs/kafl.fuzzer/pull/4)
-- robustness fixes to qemu startup (https://github.com/IntelLabs/kafl.fuzzer/pull/5)
-- Fix config loading defaults and error handling (https://github.com/IntelLabs/kafl.fuzzer/pull/9)
-- ghidra_run.sh: fail to stderr (https://github.com/IntelLabs/kafl.fuzzer/pull/11)
-- config.py: apply expand_vars only to existing config options (https://github.com/IntelLabs/kafl.fuzzer/pull/20)
-- kafl_cov.py: fix handling of timeout exception (https://github.com/IntelLabs/kafl.fuzzer/pull/21)
-- fix kafl_debug.py for new pt dump trace mode (https://github.com/IntelLabs/kafl.fuzzer/pull/26)
-- cpu affinity: consider --cpu-offset as override to auto-detection (https://github.com/IntelLabs/kafl.fuzzer/pull/27)
+- avoid Qemu hang when handling ABORT in pre-init phase (#34)
```

Comment:
Can add a link to the PR itself ?
adding `#34` isn't resolved by Github to a link to the PR unfortunately.

Reply:
seems to work? https://github.com/il-steffen/kafl.fuzzer/releases/tag/v0.1

Reply:
Oh, then it works in the release view.
I was displaying the final markdown from the diff, that's why.
LGTM